README.md

```diff
@@ -70,6 +70,11 @@ This section describes the optional enterprise functionality that can be enabled
 *  `VERBOSE`
   * Description: When set to `true` it outputs the commands and results to stdout from `configure.sh`. Otherwise, default setting is `false` and `configure.sh` is silenced.
 
+## Security 
```

Comment:
We can keep this section and get rid of the `Security` section below. 

Thanks for fixing some broken links.

Unrelated to this PR, but will be good to fix these:
- The headings, starting with `Logging` and until 
`Installing Liberty Features from local repository (19.0.0.8+)` are not right. They shouldn't be under `OpenJ9 Shared Class Cache (SCC)`. Please move them one level up (### --> ##). 
- Spell checks: replace all instances of `Decription` with `Description` and replace `Locallly` with `Locally`